azure-configurationservice/examples/test_example_configurationservice.py

```diff
@@ -0,0 +1,142 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import unittest
+from msrestazure.azure_exceptions import CloudError
+from azure.configuration import AzureConfigurationClient
+from azure.configuration import ConfigurationSetting
+
+
+class AzConfigurationClientExamples(unittest.TestCase):
+
+    def _add_for_test(self, key, label):
+        exist = bool(list(self.client.list_configuration_settings(keys=[key], labels=[label])))
+        if not exist:
+            sc = ConfigurationSetting(
+                key=key,
+                label=label,
+                value="my value",
+                content_type="my content type",
+                tags={"my tag": "my tag value"}
+            )
+            self.client.add_configuration_setting(sc)
+
+    def _delete_from_test(self, key, label):
+        try:
+            self.client.delete_configuration_setting(key=key, label=label)
+        except CloudError:
+            pass
+
+    def setUp(self):
+        super(AzConfigurationClientExamples, self).setUp()
+        # [START create_app_configuration_client]
+        import os
+        from azure.configuration import AzureConfigurationClient
+
+        connection_str = os.environ["AZ_CONFIG_CONNECTION"]
+        client = AzureConfigurationClient(connection_str)
+        # [END create_app_configuration_client]
+
+        self.client = client
+
+    def tearDown(self):
+        self._delete_from_test("MyKey", "MyLabel")
+
+    def test_azconfig_client_sample(self):
+        # This is not unit test code. Just to verify if these sample code snippets work.
```

Comment:
Could we separate each sample into its own method? it will make it easier to read

Reply:
Good question. I divided them into different methods before. But I want to make these samples to share one ConfigurationSetting instance ("MyKey", "MyLabel") so I moved to one method. I'll need to research and configure pytest and/or unittest libraries to run add_configuration_setting before other methods. I haven't known how to do it yet. Will do this later.